pyros_schemas/ros/schemagic.py

```diff
@@ -80,28 +81,24 @@ def create(ros_msg_class,
                     ros_schema_inst = RosNested(create(stype))  # we need to nest the next (Ros)Schema
 
         members.setdefault(s, ros_schema_inst)
+        schema_instance.declared_fields[s] = ros_schema_inst
```

Comment:
what are the `fields` and  `declared_fields` variables ?

Reply:
I have added the marshmallow.Schema field to force it to create a new type.